src/init.asm

```diff
@@ -126,7 +126,7 @@ init_sram_upto9:
     ; $0010 = R
 
     ; Features
-    LDA #$0015 : STA !sram_artificial_lag
+    LDA #$001B : STA !sram_artificial_lag
```

Comment:
Not what I had in mind; you don't want this change.  sram_artificial_lag gets multiplied by 16 (ASL 4) before being used as the loop variable.

I'm getting ready to pull your stuff into my branch so I can fix it then.  I just wanted to give you a heads up so you don't start defaulting the artificial lag that high before handing off to others in case the extra lag is noticeable.